processor/src/services/converters/helper.converter.ts

```diff
@@ -12,42 +12,64 @@ import {
   getProcessorUrlFromContext,
 } from '../../libs/fastify/context/context';
 
+/**
+ * Mapping logic is mainly based upon the Adyen `adyen-commercetools` connector.
+ *
+ * @param cart The CoCo cart to map the line items from
+ * @returns List of lineitems that Adyen will accept
+ */
 export const populateLineItems = (cart: Cart): LineItem[] => {
```

Comment:
probably not for this pr, but this file deserves some unit tests

Reply:
Correct and good point. I have added the tests for all of the line item mappings.